src/components/LoanPositionsTable/LoanPositionsTable.tsx

```diff
@@ -1,12 +1,11 @@
-import { BorrowPosition, CollateralPosition, LoanAsset, TickerToData } from '@interlay/interbtc-api';
 import { useId } from '@react-aria/utils';
 import { Key, ReactNode, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 
 import { convertMonetaryAmountToValueInUSD } from '@/common/utils/utils';
 import { Switch } from '@/component-library';
 import { useGetPrices } from '@/hooks/api/use-get-prices';
-import { LoanType } from '@/types/loans';
+import { BorrowPosition, CollateralPosition, LoanAsset, LoanType, TickerToData } from '@/types/loans';
```

Comment:
Imported from a custom type file for loans just because at some point we mix squid with the data coming from the lib. So if we want to keep the types consistent across the app, the loans related types should be imported from one place only (so its easy to understand)